docs/slotfilling.rst

```diff
@@ -36,13 +38,22 @@ restaurant bot, this means your initial story will look as simple as this
         - form{"name": "restaurant_form"}
         - form{"name": null}
 
-This means that even if you provide multiple slots at one, e.g. if someone says
-"I'd like a vegetarian Chinese restaurant for 8 people" the slots ``cuisine``
-and ``num_people`` won't get asked for.
+The ``FormAction`` will only requests slots which haven't already been set.
+If a user says 
+"I'd like a vegetarian Chinese restaurant for 8 people", they won't be 
+asked about the ``cuisine`` and ``num_people`` slots.
+
+Note that this works best if your slots are `unfeaturized
```

Comment:
more like, only works... otherwise you have to include the slot set events at the end 😛